tty/grlib-uart/grlib-uart.c

```diff
@@ -105,6 +111,9 @@ static struct {
 } uart_common;
 
 
+#ifdef __TARGET_RISCV64
+__attribute__((section(".interrupt"), aligned(0x1000)))
+#endif
```

Comment:
Perhaps we could make this a standard procedure? I think default ld script should handle this just fine, but not sure